src/ripple/rpc/impl/TransactionSign.cpp

```diff
@@ -1083,12 +1083,21 @@ Json::Value transactionSubmitMultiSigned (
             return RPC::make_error (rpcINVALID_PARAMS, err.str ());
         }
 
-        // The Fee field must be greater than zero.
-        if (stpTrans->getFieldAmount (sfFee) <= 0)
+        // The Fee field must be in XRP and greater than zero.
+        auto const fee = stpTrans->getFieldAmount (sfFee);
+
+        if (!isLegalNet (fee))
+        {
+            std::ostringstream err;
+            err << "Invalid " << sfFee.fieldName
+                << " field.  Fees must be specified in XRP.";
+            return RPC::make_error (rpcINVALID_PARAMS, err.str ());
+        }
+        if (fee <= 0)
```

Comment:
Gaaah.  Never mind.  Since it's multisigned I think there's always a fee associated with the signatures.  But I'm not positive.  I'll have to set up a test case and try it out.